src/App.tsx

```diff
@@ -1,18 +1,57 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext, useEffect, useRef, useState,
+} from 'react';
+import { getTodos } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Footer } from './components/Footer';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+import { Errors } from './components/Errors';
 
 export const App: React.FC = () => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState(false);
+  const [errorText] = useState('');
+  const [filter, setFilter] = useState('all');
+  const [newTodo, setNewTodo] = useState('');
+
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
 
+  const filtredTodos = todos.filter(todo => {
+    switch (filter) {
+      case Filter.ALL:
+        return todos;
+      case Filter.ACTIVE:
+        return !todo.completed;
+      case Filter.COMPLETED:
+        return todo.completed;
+      default:
+        return [];
+    }
+  });
+
+  if (error) {
+    setTimeout(() => {
+      setError(false);
+    }, 3000);
+  }
```

Comment:
you need to create setTimeout on the first render ErrorNotification and clear timeout on unmount